frontend/dockerfile/dockerfile2llb/image.go

```diff
@@ -53,6 +54,9 @@ type Image struct {
 
 	// Variant defines platform variant. To be added to OCI.
 	Variant string `json:"variant,omitempty"`
+
+	// BuildInfo defines build dependencies
+	BuildInfo source.BuildInfo `json:"moby.buildkit.buildinfo.v0,omitempty"`
```

Comment:
This should be base64. I know it makes it more hacky but better not to add big non-spec structures.